active/0003-extensible-resource-deployment-observability-improvements.md

```diff
@@ -0,0 +1,249 @@
+---
+REP Number: 0003
+Author: shenglol (Shenglong Li)
+Start Date: 2024-01-25
+Feature Status: Public Preview
+---
+
+# Extensible Resource Deployment Observability Improvements
+
+## Summary
+
+This REP outlines API enhancements for [`Microsoft.Resources/deployments`](https://learn.microsoft.com/en-us/rest/api/resources/deployments/create-or-update?view=rest-resources-2021-04-01&tabs=HTTP) and [`Microsoft.Resources/deployments/operations`](https://learn.microsoft.com/en-us/rest/api/resources/deployment-operations/get-at-scope?view=rest-resources-2021-04-01) to empower users to effectively monitor the progress of extensible resource deployments through Azure CLI, Azure Powershell, and Azure Portal deployments blade, and other relavant Azure Resource Manager interfaces.
+
+## Terms and definitions
+
+### Deployment Provider
+
+An API abstraction for the following operations:
+
+- Creating or updating Azure Resources or extensible resources declared in Bicep files or an ARM templates;
```

Comment:
- Creating or updating Azure Resources or extensible resources declared in Bicep files or ARM templates;
Just a tiny grammar nitpick - to take out "an" before ARM Templates